src/components/Form/SearchWrapper.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import T from 'prop-types'
+import { randomId, classNames } from '../../utils/'
+
+const initial = 'c-search'
+
+const SearchWrapper = ({ id, className, submitPosition, ...rest }) => (
```

Comment:
we want a `handleSubmit`callback and `onChange` callback so consumers can tap into events when they change.
Think about what arguments you should pass to these...

Reply:
`onChange` event gets passed up from `Form.TextInput`... so if that doesn't have an `onChage` callback already then you should add one to that.

Reply:
done